<!DOCTYPE html>
<html lang="en-us">

<head>
  <meta http-equiv="X-Clacks-Overhead" content="GNU Terry Pratchett" />
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>Manuscript | Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver | Alma Andersson</title>
<meta name="title" content="Manuscript | Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver" />
<meta name="description" content="I&rsquo;m thrilled to announce that a collaboration with my good friend and colleague, Franziska Hildebrandt, now has matured to the state of being ready for journal submission, which also coincides with a pre-print upload to bioRxiv.
The pre-print is titled &ldquo;Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver&rdquo;, and focus on the application of ST2K (similar to the old legacy ST1K and the newer Visium platform)." />
<meta name="keywords" content="" />


<meta property="og:url" content="https://almaan.github.io/blog/2021-01-12-liver-biorxiv/">
  <meta property="og:site_name" content="Alma Andersson">
  <meta property="og:title" content="Manuscript | Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver">
  <meta property="og:description" content="I’m thrilled to announce that a collaboration with my good friend and colleague, Franziska Hildebrandt, now has matured to the state of being ready for journal submission, which also coincides with a pre-print upload to bioRxiv.
The pre-print is titled “Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver”, and focus on the application of ST2K (similar to the old legacy ST1K and the newer Visium platform).">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="blog">
    <meta property="article:published_time" content="2021-01-12T11:59:19+01:00">
    <meta property="article:modified_time" content="2021-01-12T11:59:19+01:00">
    <meta property="og:image" content="https://almaan.github.io/blog/2021-01-12-liver-biorxiv/feature-by-distance.png">




  <meta name="twitter:card" content="summary_large_image">
  <meta name="twitter:image" content="https://almaan.github.io/blog/2021-01-12-liver-biorxiv/feature-by-distance.png">
  <meta name="twitter:title" content="Manuscript | Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver">
  <meta name="twitter:description" content="I’m thrilled to announce that a collaboration with my good friend and colleague, Franziska Hildebrandt, now has matured to the state of being ready for journal submission, which also coincides with a pre-print upload to bioRxiv.
The pre-print is titled “Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver”, and focus on the application of ST2K (similar to the old legacy ST1K and the newer Visium platform).">




  <meta itemprop="name" content="Manuscript | Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver">
  <meta itemprop="description" content="I’m thrilled to announce that a collaboration with my good friend and colleague, Franziska Hildebrandt, now has matured to the state of being ready for journal submission, which also coincides with a pre-print upload to bioRxiv.
The pre-print is titled “Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver”, and focus on the application of ST2K (similar to the old legacy ST1K and the newer Visium platform).">
  <meta itemprop="datePublished" content="2021-01-12T11:59:19+01:00">
  <meta itemprop="dateModified" content="2021-01-12T11:59:19+01:00">
  <meta itemprop="wordCount" content="694">
  <meta itemprop="image" content="https://almaan.github.io/blog/2021-01-12-liver-biorxiv/feature-by-distance.png">
<meta name="referrer" content="no-referrer-when-downgrade" />

  <style>
  body {
    font-family: Verdana, sans-serif;
    margin: auto;
    padding: 20px;
    max-width: 720px;
    text-align: left;
    background-color: #fff;
    word-wrap: break-word;
    overflow-wrap: break-word;
    line-height: 1.5;
    color: #444;
  }

  h1,
  h2,
  h3,
  h4,
  h5,
  h6,
  strong,
  b {
    color: #222;
  }

  a {
    color: #3273dc;
     
  }

  .title {
    text-decoration: none;
    border: 0;
  }

  .title span {
    font-weight: 400;
  }

  nav a {
    margin-right: 10px;
  }

  textarea {
    width: 100%;
    font-size: 16px;
  }

  input {
    font-size: 16px;
  }

  content {
    line-height: 1.6;
  }

  table {
    width: 100%;
  }

  img {
    max-width: 100%;
  }

  code {
    padding: 2px 5px;
    background-color: #f2f2f2;
  }

  pre code {
    color: #222;
    display: block;
    padding: 20px;
    white-space: pre-wrap;
    font-size: 14px;
    overflow-x: auto;
  }

  div.highlight pre {
    background-color: initial;
    color: initial;
  }

  div.highlight code {
    background-color: unset;
    color: unset;
  }

  blockquote {
    border-left: 1px solid #999;
    color: #222;
    padding-left: 20px;
    font-style: italic;
  }

  footer {
    padding: 25px;
    text-align: center;
  }

  .helptext {
    color: #777;
    font-size: small;
  }

  .errorlist {
    color: #eba613;
    font-size: small;
  }

   
  ul.blog-posts {
    list-style-type: none;
    padding: unset;
  }

  ul.blog-posts li {
    display: flex;
  }

  ul.blog-posts li span {
    flex: 0 0 130px;
  }

  ul.blog-posts li a:visited {
    color: #8b6fcb;
  }
</style>

    <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
<script>
  MathJax = {
    tex: {
      displayMath: [['\\[', '\\]'], ['$$', '$$']],  
      inlineMath: [['\\(', '\\)'],['$','$']]                  
    }
  };
</script>

  

  
</head>

<body>
  <header><a href="/" class="title">
  <h2>Alma Andersson</h2>
</a>
<nav><a href="/">Home</a>

<a href="/publications/">Publications</a>


<a href="/blog">Blog</a>

</nav>
</header>
  <main>

<h1>Manuscript | Spatial Transcriptomics to define transcriptional patterns of zonation and structural components in the liver</h1>
<p>
  <i>
    <time datetime='2021-01-12' pubdate>
      12 Jan, 2021
    </time>
  </i>
</p>

<content>
  <p>I&rsquo;m thrilled to announce that a collaboration with my good friend and colleague,
<a href="https://orcid.org/0000-0002-2673-1704">Franziska Hildebrandt</a>, now has matured
to the state of being ready for journal submission, which also coincides with a
pre-print upload to
<a href="https://www.biorxiv.org/content/10.1101/2021.01.11.426100v1">bioRxiv</a>.</p>
<p>The pre-print is titled <em>&ldquo;Spatial Transcriptomics to define transcriptional
patterns of zonation and structural components in the liver&rdquo;</em>, and focus on the
application of ST2K (similar to the old legacy ST1K and the newer Visium
platform). It is the first time that ST (Spatial Transcriptomics) has been
applied to the mouse liver, and aims to show how the technique enables the liver
to be explored from new, different and illuminating perspective. A lot of
standard analysis, such as clustering and identification of differential genes
(DEGs) are presented in the paper and show affirmative results, attesting to the
(experimental) method&rsquo;s ability to delineate regions with various expression
profiles (often overlapping well with morphological structures).</p>
<p>My contribution to this paper was nevertheless slightly different and consisted mainly of two parts:</p>
<ul>
<li>Construction of a framework to model feature signals as a function of the distance to a given object.</li>
<li>Development of a classifier to predict <em>vein type</em> (a binary classification
task) based on the expression profiles surrounding a certain genes.</li>
</ul>
<p>Both concepts are excellent examples of analyses that are exclusive to spatial
data, i.e., similar information cannot be obtained from single cell experiments;
something I&rsquo;m keen to highlight, as it is often the case that one fails to make
proper use of the &ldquo;spatial&rdquo; part in spatial transcriptomics data, and rather treat it as single
cell data, to then visualize the results spatially. Both implementations are very
simple, but I have not seen similar analyses in ST or Visium data yet, of course
that could be due to me not having surveyed the field thoroughly enough, but
still. Allow me to elaborate some on these two concepts below.</p>
<h3 id="feature-by-distance">Feature by distance</h3>
<p>In the liver tissue one can broadly classify vein structures as either <em>portal</em>
or <em>central</em> based on their position and connections. Previous studies have
shown that these structures are associated with slightly different expression
profiles, as different celltypes (periportal respective pericentral
hepatocytes) are present in them.</p>
<p>What we did in this study was to first &ldquo;measure&rdquo; the distance of each spot
(spatial capture location) to the nearest vein (of respective type), and then
for each gene create a set of distance-expression tuples that could be
visualized in a scatter plot. While these values are discrete, once a curve
smooting techniques is applied we can easily think of the resulting curve as a
function that maps <em>distance to gene expression</em>. By doing so one could
really see how the associated marker genes (to each vein) had a high expression
near their respective vein, and how this decreased as distance increased
(inverse relationship). You can see some examples of these trends in the plots
below, top row displays portal vein marker genes, bottom central:</p>
<p><img src="feature-by-distance.png" alt="feature by distance"></p>
<h3 id="classification-of-vein-types-by-expression">Classification of vein types by expression</h3>
<p>Now, while pathologists and experts in the field usually are able to annotate
each vein type by visual inspection (this is how we obtained the annotations
used for the feature by distance plots), there are sometimes ambiguities and
limited access to such individuals. However, the expression profiles of portal
and central veins should be the similar for all veins of the same type,
independent of whether identity is easy to deduce from morphology or not.
Hence, by creating <em>neighborhood expression profiles</em> (NEPs) surrounding each
vein, and training a classifier (actually logistic regression) using NEPs from
veins with known identities; we were able to predict vein type of ambiguous
veins. During cross-validation (leave one <em>sample</em> out) the model also performed
very well with higher than 80% accuracy.</p>
<p><img src="class.png" alt="classification"></p>
<p>To read more about the study, check out the manuscript
<a href="https://www.biorxiv.org/content/10.1101/2021.01.11.426100v1">here</a>, and the
Python package we released (<em>hepaquery</em>) to perform similar analyses with any
type of spatial transcriptomics data at the
<a href="https://github.com/almaan/ST-mLiver">github</a> page.</p>
<p>I&rsquo;m very happy to have worked with a nice team with complementary
biology and computational skills, this really resulted in a neat manuscript;
also a big kudos to the first author Franziska, who pushed very hard in the end
to finalize this project.</p>

</content>
<p>
  
</p>

  </main>
  <footer>Made with <a href="https://github.com/janraasch/hugo-bearblog/">Hugo ʕ•ᴥ•ʔ Bear</a>
</footer>

    
</body>

</html>
