<!DOCTYPE html>
<html lang="en-us">

<head><script src="/livereload.js?mindelay=10&amp;v=2&amp;port=1313&amp;path=livereload" data-no-instant defer></script>
  <meta http-equiv="X-Clacks-Overhead" content="GNU Terry Pratchett" />
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
<title>What is thy name? | Alma Andersson</title>
<meta name="title" content="What is thy name?" />
<meta name="description" content="A brief discussion about the naming convention of the observations in spatial transcriptomics." />
<meta name="keywords" content="reflections," />


<meta property="og:url" content="http://localhost:1313/blog/2022-05-06-capture-locations/">
  <meta property="og:site_name" content="Alma Andersson">
  <meta property="og:title" content="What is thy name?">
  <meta property="og:description" content="A brief discussion about the naming convention of the observations in spatial transcriptomics.">
  <meta property="og:locale" content="en_us">
  <meta property="og:type" content="article">
    <meta property="article:section" content="blog">
    <meta property="article:published_time" content="2022-05-06T06:01:05+01:00">
    <meta property="article:modified_time" content="2022-05-06T06:01:05+01:00">
    <meta property="article:tag" content="Reflections">




  <meta name="twitter:card" content="summary">
  <meta name="twitter:title" content="What is thy name?">
  <meta name="twitter:description" content="A brief discussion about the naming convention of the observations in spatial transcriptomics.">




  <meta itemprop="name" content="What is thy name?">
  <meta itemprop="description" content="A brief discussion about the naming convention of the observations in spatial transcriptomics.">
  <meta itemprop="datePublished" content="2022-05-06T06:01:05+01:00">
  <meta itemprop="dateModified" content="2022-05-06T06:01:05+01:00">
  <meta itemprop="wordCount" content="1196">
  <meta itemprop="keywords" content="Reflections">
<meta name="referrer" content="no-referrer-when-downgrade" />

  <style>
  body {
    font-family: Verdana, sans-serif;
    margin: auto;
    padding: 20px;
    max-width: 720px;
    text-align: left;
    background-color: #fff;
    word-wrap: break-word;
    overflow-wrap: break-word;
    line-height: 1.5;
    color: #444;
  }

  h1,
  h2,
  h3,
  h4,
  h5,
  h6,
  strong,
  b {
    color: #222;
  }

  a {
    color: #3273dc;
     
  }

  .title {
    text-decoration: none;
    border: 0;
  }

  .title span {
    font-weight: 400;
  }

  nav a {
    margin-right: 10px;
  }

  textarea {
    width: 100%;
    font-size: 16px;
  }

  input {
    font-size: 16px;
  }

  content {
    line-height: 1.6;
  }

  table {
    width: 100%;
  }

  img {
    max-width: 100%;
  }

  code {
    padding: 2px 5px;
    background-color: #f2f2f2;
  }

  pre code {
    color: #222;
    display: block;
    padding: 20px;
    white-space: pre-wrap;
    font-size: 14px;
    overflow-x: auto;
  }

  div.highlight pre {
    background-color: initial;
    color: initial;
  }

  div.highlight code {
    background-color: unset;
    color: unset;
  }

  blockquote {
    border-left: 1px solid #999;
    color: #222;
    padding-left: 20px;
    font-style: italic;
  }

  footer {
    padding: 25px;
    text-align: center;
  }

  .helptext {
    color: #777;
    font-size: small;
  }

  .errorlist {
    color: #eba613;
    font-size: small;
  }

   
  ul.blog-posts {
    list-style-type: none;
    padding: unset;
  }

  ul.blog-posts li {
    display: flex;
  }

  ul.blog-posts li span {
    flex: 0 0 130px;
  }

  ul.blog-posts li a:visited {
    color: #8b6fcb;
  }
</style>

    <script id="MathJax-script" async src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml.js"></script>
<script>
  MathJax = {
    tex: {
      displayMath: [['\\[', '\\]'], ['$$', '$$']],  
      inlineMath: [['\\(', '\\)']]                  
    }
  };
</script>

  

  
</head>

<body>
  <header><a href="/" class="title">
  <h2>Alma Andersson</h2>
</a>
<nav><a href="/">Home</a>

<a href="/about/">About Me</a>


<a href="/blog">Blog</a>

</nav>
</header>
  <main>

<h1>What is thy name?</h1>
<p>
  <i>
    <time datetime='2022-05-06' pubdate>
      06 May, 2022
    </time>
  </i>
</p>

<content>
  <p>You know that feeling when you&rsquo;re frustrated over something that has absolutely
no real importance, but you just can&rsquo;t let it go. That&rsquo;s me right now. <!-- raw HTML omitted -->I&rsquo;m seriously bothered by the fact that every single
publication seems to use a different term to denote the locations in
spatial transcriptomics data.<!-- raw HTML omitted --></p>
<p>It&rsquo;s not that people aren&rsquo;t using <em>my favorite</em> term (FYI, I don&rsquo;t have one)
that annoys me, but the fact that there&rsquo;s no consensus on which term to use.
There are <em>beads, cells, features, voxels, pixels, spots, capture locations</em>,
and many other innovative alternatives. The current practice is <strong>confusing</strong>,
and settling on one term should not be that hard.</p>
<blockquote>
<p>&ldquo;If names are not correct, language will not be in accordance with the truth of things.&rdquo;
&ndash; Confucius</p>
</blockquote>
<p>My idea with this post was to present some of the different terms I&rsquo;ve
encountered so far (disclaimer: this is not an exhaustive list**, together with a
bit of personal commentary - mainly to provide some context. To then couple this
with a poll of some sort, and see what the thoughts of the community are.
Perhaps this could spark some sort of discussion or reflection on the topic.</p>
<p><strong>NOTE</strong> : this list was updated (2022-05-09) after posting
<a href="https://twitter.com/aalmaander/status/1523290638422188034?s=20&amp;t=04LQK9AF-yQHLcgYT_K2dA">this</a>
twitter post, where some other suggestions were presented. Thanks to all the
people who contributed to a really nice discussion!</p>
<ul>
<li>
<p><strong>Bead</strong> : popularized by
<a href="https://www.science.org/doi/10.1126/science.aaw1219">Slide-seq</a>, in which
beads are used to capture the transcripts within the tissue. This also
resonates well with data from the HDST platform. However, it makes zero sense
with data from other platforms, like Visium, where capture probes are
immobilized to a solid surface, nor to imaging-based techniques like MERFISH or
SeqFISH where single molecules are visualized and there&rsquo;s no &ldquo;capture&rdquo; going
on.</p>
</li>
<li>
<p><strong>Spot</strong> : is used somewhat ambiguously in different techniques. In the
<a href="https://www.science.org/doi/10.1126/science.aaf2403">ST</a> and
<a href="https://www.10xgenomics.com/products/spatial-gene-expression">Visium</a>
platform, the mRNA capture locations are printed onto the surface in a fashion
similar to how
<a href="https://www.nature.com/scitable/definition/microarray-202/">microarrays</a> were
constructed, which made the use of &ldquo;spot&rdquo; to describe the capture locations
quite natural. But just as &ldquo;beads&rdquo; makes little sense to describe the capture
locations in Visium data, &ldquo;spots&rdquo; makes no sense in Slide-seq data. What adds
another layer of confusion is that in <a href="https://www.nature.com/articles/s41598-019-43943-8">imaging-based
methods</a>, &ldquo;spot&rdquo; is
sometimes used to describe the signal from a single molecule. This kind of
ambiguity adds even more confusion to the already confusing situation. &ldquo;Spot&rdquo;
is currently (2022-05-06) favored by 10x Genomics (the distributor of Visium).
10x Genomics has a large reach and, i.e., whatever term they chose, a lot of
people will use it.</p>
</li>
<li>
<p><strong>Feature</strong> : have been used by some authors to describe the spatial
locations, for example, <a href="https://doi.org/10.1016/j.molcel.2021.03.016">this</a>
review. Without bashing too hard on the people who prefer this term, I must
confess that I find it extremely impractical to use the word &ldquo;feature&rdquo; to
describe the &ldquo;observations&rdquo;. To me, a feature is the information associated
with the spatial location (i.e., gene expression). Also, most analysis
frameworks (e.g., Seurat) tend to call the genes (if you&rsquo;re analyzing gene
expression data) features. If we were to use the term feature also for our
spatial locations, would we then say that we have a &ldquo;feature times feature&rdquo;
matrix? That&rsquo;s not very helpful. If confusion is something we want to avoid,
we should not pursue the use of this term.</p>
</li>
<li>
<p><strong>Pixel</strong> : a term that most of us are familiar with, but usually in the
context of pictures or camera lenses. The word pixel is a portmanteau of the
two words &ldquo;picture&rdquo; and &ldquo;element&rdquo; (where &ldquo;pix&rdquo; is used as short form of
&ldquo;picture&rdquo;). I first saw this being used in the <a href="https://www.nature.com/articles/s41587-021-00830-w">RCTD
paper</a>, but have also
heard other people use the term in presentations. To some extent, this is a
slightly more general term than &ldquo;bead&rdquo; or &ldquo;spot&rdquo;, and it works with
imaging-based spatial transcriptomics methods. Also, to some extent, you could
consider the Visium or Slide-seq data as an &ldquo;image&rdquo; with ~20,000 channels (one
channel - one gene) instead of the standard three channels (RGB), but that&rsquo;s a
bit of a stretch. Calling the spatial data a &ldquo;picture&rdquo; when it&rsquo;s not really
image data that we&rsquo;re working does not really vibe with me.</p>
</li>
<li>
<p><strong>Voxel</strong> : is a lesser known relative to the pixel, where &ldquo;vo&rdquo; for <em>volume</em>
has replaced &ldquo;pix&rdquo; for picture; giving us a <em>&ldquo;volume element&rdquo;</em>. My first
exposure to the usage of this term to denote spatial locations was the
<a href="https://www.nature.com/articles/s41592-021-01264-7">Tangram paper</a>. I&rsquo;m not
fully sure why this term was preferred over pixel, but perhaps &ldquo;volume&rdquo; was
considered a more general term that could describe elements in both 2D and 3D
methods (if you consider area as a &ldquo;2D volume&rdquo;). Although, I believe, for most
people, volume relates very much to 3D objects; thus, potentially puzzling
some people.</p>
</li>
<li>
<p><strong>Spatial location</strong> : sometimes, often in an attempt to be method agnostic, I
(and others) have used this term instead of any of the aforementioned
alternatives. It definitely does the trick, it states exactly what we&rsquo;re
dealing with and there&rsquo;s no room for confusion. But it&rsquo;s terribly <del>unsexy</del>
boring, as well as being quite long (two words!) and inconvenient to use.
Sure, one could go for yet another portmanteau, but referring to spatial locations
as <em>&ldquo;splocs&rdquo;</em> sounds&hellip; ehm&hellip; ridiculous.</p>
</li>
<li>
<p><strong>Gexel</strong> : a what? Yeah you read that correct, a <em>gexel</em>. I stumbled upon
this term when reading the 2021 <a href="https://www.sciencedirect.com/science/article/pii/S2405471221001514">MULTILAYER
paper</a>,
where the authors got inspired by the pixels and conceived (I could not find
any earlier use) of the &ldquo;gexel&rdquo;, being short for <strong>&ldquo;gene expression
element&rdquo;</strong>. At first I kind of frowned at this term, but the more I&rsquo;ve been
thinking about it, the more I&rsquo;ve come to like it. It&rsquo;s fully method agnostic,
it&rsquo;s short and convenient to use, and &ldquo;gene expression element&rdquo; captures the
essence of what our spatial locations are. The big downside to this term is
its lack of <em>generalization to other modalities</em>, or multimodal assays where
different kinds of information are registered at the same location. Perhaps
one could use <em>daxel</em> for &ldquo;data element&rdquo; or <em>spexel</em> for &ldquo;spatial expression
element&rdquo;, but both sound a bit &ldquo;forced&rdquo;.</p>
</li>
</ul>
<p>We can summarize this is a very basic table:</p>
<table>
<thead>
<tr>
<th>Term</th>
<th>Agnostic</th>
<th>Descriptive</th>
<th>Unambiguous</th>
<th>Convenient</th>
<th>Multimodal</th>
</tr>
</thead>
<tbody>
<tr>
<td>bead</td>
<td>no</td>
<td>no</td>
<td>yes</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>spot</td>
<td>no</td>
<td>no</td>
<td>no</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>feature</td>
<td>yes</td>
<td>no</td>
<td>no</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>pixel</td>
<td>yes</td>
<td>no</td>
<td>yes</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>voxel</td>
<td>yes</td>
<td>no</td>
<td>yes</td>
<td>yes</td>
<td>yes</td>
</tr>
<tr>
<td>spatial location</td>
<td>yes</td>
<td>yes</td>
<td>yes</td>
<td>no</td>
<td>yes</td>
</tr>
<tr>
<td>gexel</td>
<td>yes</td>
<td>yes (+)</td>
<td>yes</td>
<td>yes</td>
<td>no</td>
</tr>
</tbody>
</table>
<p>Columns: Agnostic - method agnostic, Descriptive - conveys information about what it
represents, Unambiguous - will not be confused with other elements of the data,
Convenient - easy to use and say, Multimodal - can be used in mulitmodal
assays.</p>
<p>P.S. I&rsquo;m acutely aware of the fact that <em>&ldquo;being bothered by the inconsistency in
naming of spatial locations&rdquo;</em> would be the perfect answer to <em>&ldquo;tell me you have
no real problems without saying you have no real problems.&rdquo;</em> Other <em>more
important</em> things are happening around the world, I acknowledge that 100%, but I
don&rsquo;t want those things to engulf my whole existence. Silly discussions must
also be allowed, just as happiness must be allowed space in times of grief and
sorrow.</p>

</content>
<p>
  
  <a href="http://localhost:1313/tags/reflections/">#Reflections</a>
  
</p>

  </main>
  <footer>Made with <a href="https://github.com/janraasch/hugo-bearblog/">Hugo ʕ•ᴥ•ʔ Bear</a>
</footer>

    
</body>

</html>
